apis/elbv2/v1beta1/targetgroupbinding_types.go

```diff
@@ -145,6 +145,10 @@ type TargetGroupBindingSpec struct {
 	// ipAddressType specifies whether the target group is of type IPv4 or IPv6. If unspecified, it will be automatically inferred.
 	// +optional
 	IPAddressType *TargetGroupIPAddressType `json:"ipAddressType,omitempty"`
+
+	// VpcId is the VPC of the TargetGroup. If unspecified, it will be automatically inferred.
+	// +optional
+	VpcId string `json:"vpcId,omitempty"`
```

Comment:
we follow the kubernetes API convention in our CRD to use same case for abbreviation words. 

so this should be `vpcID` instead of `vpcId` (btw, i know AWS API uses `vpcId`).